tyson/README.md

```diff
@@ -1,26 +1,68 @@
 # TySON 🥊
-### Use TypeScript as a configuration language
+### TypeScript as a Config Language
 
 # What is it?
 TySON (TypeScript Object Notation) is a subset of TypeScript, chosen to be useful as an embeddable configuration
-language that generates JSON. You can think of it as JSON + types + functions using
+language that generates JSON.
+You can think of TySON as **JSON + comments + types + basic logic** using
 TypeScript syntax. TySON files use the `.tson` extension.
 
-The goal is to make it possible for all major programming languages to read
```

Comment:
I don't want to fully get rid of this sentence. Since TySON is in some ways just TypeScript, I think an objection to it can be "why do you even need TySON, just use TypeScript directly" – the idea of making it accessible from several host languages without a direct dependency in the JS ecosystem is valuable.

Is there a different place where we could keep this sentence? Or something similar?

Reply:
Oh, did I delete this sentence? That was by mistake. Adding it back...